package.json

```diff
@@ -1272,21 +1269,21 @@
     "mocha": "^4.1.0",
     "module-alias": "^2.2.1",
     "sinon": "~5.0.7",
-    "ts-loader": "^6.0.4",
-    "ts-node": "^6.0.0",
+    "ts-loader": "^6.2.2",
+    "ts-node": "^8.8.1",
```

Comment:
This new version of `ts-node` seems to be causing the build to fail on linux.  Can you please fix it or revert the change?

Reply:
I saw your update, but the build still fails.  Unfortunately, I don't think this was fixed.

Reply:
Yes I just saw it fails. I am trying reverting next.